albumentations/augmentations/functional.py

```diff
@@ -6,7 +6,9 @@
 from itertools import product
 import cv2
 import numpy as np
+import skimage
```

Comment:
I think it is better to explicitly define `scikit-image` in `install_requires` in `setup.py` since we plan to use it after we remove imgaug as a dependency. - https://github.com/albumentations-team/albumentations/blob/master/setup.py#L8.

Reply:
But it is already in the dependencies, as you can see from your link. Or what you mean?

Reply:
Sorry, my bad. I somehow managed to miss it.